### ISSUE TITLE
Killing a loop execution kills Gantt and future flow executions

### ISSUE THREAD

**Issue by smunteankestra** [kestra-io/kestra#4510]
### Describe the issue

Step 1: Run the flow below for a couple of seconds
Step 2: Kill the Execution
Step 3: Start the execution with 1 loop instead of 800

Expected - to run execution successfully
Actual - Execution is stuck  

Flow:
```yaml
id: chucknorris
namespace: chucknorris.chucknorris.chucknorris.chucknorris.chucknorris
description: Save and Execute the flow

labels:
  env: dev
  project: myproject
tasks:
  - id: loop_jokes
    type: io.kestra.plugin.core.flow.EachParallel
    description: "Loop to request random jokes 200 times."
    value: |
      {{ range(1, 801) }}
    tasks:
      - id: timeout_check
        type: io.kestra.plugin.core.http.Request
        uri: https://api.chucknorris.io/jokes/random
        method: GET
        contentType: application/json
        timeout: PT20S
```

### Environment

- Kestra Version: 1.8
- Operating System (Docker):
- Java Version (Docker):

**Comment by anna-geller** [comment]
https://share.descript.com/view/xbr2Z8WGltn

**Comment by loicmathieu** [comment]
Tested on latest develop and I can open the page, I know there were improvements in this area.

However, yes, this can occur as here you are launching 800 tasks in parallel which will take all resources of your local computer and will consume a lot of resources to display the Gantt and the Log page in your browser, normally the improvements on the Gantt and log pages to compute only the displayable part will help but keep in mind that we advise to not have more than a few hundreds of taskrun for an execution so 800 is way more than what we advise.

So I'm closing it, my advise would be to always set a concurrency to a sane level (our default is wrong, I know, it should not be un-limitted concurrency by default, I already advocated to change that, but we didn't go to an agreement on this).
